code/datums/outfits/outfit_admin.dm

```diff
@@ -1385,3 +1385,42 @@
 	mask = /obj/item/clothing/mask/breath/vox
 	belt = /obj/item/tank/internals/emergency_oxygen/double/vox
 	box = /obj/item/storage/box/survival_vox
+
+/datum/outfit/admin/enforcer
+	name = "Oblivion Enforcer"
+
+	uniform = /obj/item/clothing/under/color/white
+	shoes = /obj/item/clothing/shoes/white/enforcer
+	back = /obj/item/storage/backpack/satchel
+	id = /obj/item/card/id
+	//The hood on this gets enabled on the after-equip proc.
+	suit = /obj/item/clothing/suit/hooded/oblivion
+	gloves = /obj/item/clothing/gloves/color/white/supermatter_immune
+	mask = /obj/item/clothing/mask/gas/voice_modulator/oblivion
+	l_ear = /obj/item/radio/headset
+	suit_store = /obj/item/supermatter_halberd
+	r_pocket = /obj/item/tank/internals/emergency_oxygen/double
+	box = /obj/item/storage/box/wizard
+
+	//The spells that the enforcer has.
+	var/list/spell_paths = list(/obj/effect/proc_holder/spell/aoe/conjure/summon_supermatter, /obj/effect/proc_holder/spell/ethereal_jaunt, \
```

Comment:
Why "\\" here?

Reply:
don't you have to do that to split a list across multiple lines?

Reply:
I don't think we need it, you can confirm with someone else too.

Reply:
\s arent needed unless you're in a define really